test/suite/wttest.py

```diff
@@ -749,6 +760,15 @@ def ttyVerbose(level, message):
     def shortid(self):
         return self.id().replace("__main__.","")
 
+    def sanitized_shortid(self):
+        """
+        Return a name that is suitable for creating file system names.
+        In particular, names with scenarios look like
+        'test_file.test_file.test_funcname(scen1.scen2.scen3)'.
+        So transform '(', but remove final ')'.
+        """
+        return self.shortid().translate(str.maketrans('($[]/ ','______', ')'))
```

Comment:
I'm curious about the reason there are multiple underscore characters in the 2nd str argument.

Reply:
I think I understand -`maketrans()` requires one replacement character to be listed for each character to be replaced in the str argument.

Reply:
Yes, that's it exactly.